source/direct_debit/index.html.md.erb

```diff
@@ -257,23 +256,31 @@ This would return payments 501 to 1000.
     </tr>
     <tr>
     <td nowrap><code>started</code></td>
-    <td>The user has visited the <code>next_url</code> and landed on the GOV.UK Pay page to enter their account information.</td>
+    <td>Your user has visited the <code>next_url</code> and landed on the GOV.UK Pay page to enter their account information.</td>
+    </tr>
+    <tr>
+    <td nowrap><code>abandoned</code></td>
+    <td>GOV.UK Pay did not set up the mandate, because your user took longer than 90 minutes to finish entering their account information.</td>
```

Comment:
It could also be because the user explicitly cancelled (including if they chose to cancel after answering no to the question about whether they’re authorised to set up Direct Debits on the account).

Reply:
Good to know, thanks - I've added this in https://github.com/alphagov/pay-tech-docs/pull/306/commits/dcd457b6ebf4f546e8be6da060b56410a816b0c7.